pkg/util/iptables/iptables.go

```diff
@@ -783,3 +789,90 @@ func isResourceError(err error) bool {
 	}
 	return false
 }
+
+// ParseError records the payload when iptables restore fails
+type ParseError interface {
+	// Line returns the number of lines on which iptables-restore is failing
```

Comment:
nitpick: something like "Line returns the line number on which the parse error was reported."